core/jobs/src/test/java/com/cognifide/aet/job/common/modifiers/header/HeaderModifierTest.java

```diff
@@ -58,7 +58,7 @@ public void testCollect() throws Exception {
     headerModifier.setParameters(ImmutableMap.of("key", "header", "value", "value1"));
     headerModifier.collect();
 
-    verify(proxyServer).addHeader("header", "value1");
+    verify(proxyServer).addHeader("header", "value1", false);
```

Comment:
Is there any way to test the `override` approach?
Maybe if not the unit test, some integration test (e.g. page displaying `User-Agent` header value?)